doc/quick-install.md

```diff
@@ -1,14 +1,12 @@
 # Jitsi Meet quick install
 
-This document describes the required steps for a quick Jitsi Meet installation on a Debian based GNU/Linux system. Debian 8 (Jessie) or later, and Ubuntu 14.04 or later are supported out-of-the-box.
-
-Also note that a recent default Ubuntu installation has only the `main` repository enabled, and Jitsi Meet needs packages from `universe`. Check your `/etc/apt/sources.list` file, and if `universe` is not present refer to [Ubuntu's documentation](https://help.ubuntu.com/community/Repositories/Ubuntu) on how to enable it. (Usually it amounts to copying the `main` lines and changing to `universe`.)
+This guide helps you  ___host your own Jitsi server___. If you want to have a video conference without setting up any infrastructure, use https://meet.jit.si instead.
 
-N.B.:
+This document describes the required steps for a quick Jitsi Meet installation on a Debian based GNU/Linux system. Debian 8 (Jessie) or later, and Ubuntu 14.04 or later are supported out-of-the-box.
 
```

Comment:
These I think are not relevant anymore. I would say Ubuntu 18.04 , and Debian 9.

Reply:
14.04 for sure is not working because of the systemd requirement. Debian 8 is end of life.

Reply:
Are you sure Debian 8 has reached EOL?  It remains listed as LTS on this page: https://wiki.debian.org/LTS

+1 on switching Ubuntu to 18.04 (current LTS)

Reply:
Well it ends in few months :) June 30, 2020 so I would say let's remove it since we are here

Reply:
That seems reasonable; https://wiki.debian.org/DebianReleases was useful to confirm that Debian 9 EOL in June 2020 should meet their LTS target of 5 years for stable releases.